flip.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @module 101/flip
+ */
+
+/**
+ * Returns a function with flipped arguments
+ * @function module:101/flip
+ * @param {function} f - function to be flipped
+ * @return {function}
+ */
+module.exports = flip;
+
+function flip(f) {
+  return function() {
+    var args = [].slice.call(arguments);
```

Comment:
Yup, for consistency. Side note here is that using the prototype does not create a new array instance just for the purposes of using slice.